Content search

borna223 opened the pull request:
Imported content can now be filtered by user.

Commit:
Add content search

Commit:
Improve UX

Commit:
Fuzzy test repo names

Commit:
Merge branch 'develop' into feature/content-search

Commit:
Replace fuzzy with fuzzysearch

vladimyr commented on client/admin/components/Program/Content/index.vue:
```diff
@@ -67,9 +67,10 @@ export default {
+      if (!pattern) return this.importedRepos;
       return filter(this.importedRepos, it => {
-        return it.name && fuzzy.test(this.filter, it.name);
+        return it.name && fuzzysearch(pattern, it.name.toLowerCase());
```
```js
const fuzzy = (needle, haystack) => fuzzysearch(needle.toLowerCase(), haystack.toLowerCase());
// and later on...
return it.name && fuzzy(this.filter, it.name);
```

vladimyr commented:
@borna223 Resolve `package-lock.json` conflicts...

Commit:
Move fuzzysearch to helper function

Commit:
Merge branch 'develop' into feature/content-search

vladimyr committed:
Cleanup :lipstick:

- disabling search if there are no imported repos
- extracting `.table-toolbar` helper to global stylesheet
- extracting table row definitions outside table components
